### PR TITLE
FT(kinetic) - switch the library in es5

### PR DESCRIPTION
**Pull request by AntoninCoulibaly**
It is simpler to have a library in ES5 for avoiding conflicts with babel

### CODE REVIEW

File: lib/kinetic.js [hunk 1]
```diff
@@ -774,10 +781,11 @@ export class SetupResponsePDU extends PDU {
  *                  {number} [options.clusterVersion = 0] - the cluster version
  * @returns {Kinetic} - message structure following the kinetic protocol
  */
-export class NoOpPDU extends PDU {
-    constructor(sequence, options = {}) {
+class NoOpPDU extends PDU {
+    constructor(sequence, optionsA) {
         super();
 
+        const options = optionsA ? optionsA : {};
```
Comment (michael-zapata) [inline lib/kinetic.js]: `options = optionsA || {}`

File: lib/kinetic.js [hunk 2]
```diff
@@ -841,10 +849,12 @@ export class NoOpResponsePDU extends PDU {
  *                  the operation.
  * @returns {Kinetic} - message structure following the kinetic protocol
  */
-export class PutPDU extends PDU {
-    constructor(sequence, keyArg, chunkSize, options = {}) {
+class PutPDU extends PDU {
+    constructor(sequence, keyArg, chunkSize, optionsA) {
         super();
 
+        const options = optionsA ? optionsA : {};
```
Comment (michael-zapata) [inline lib/kinetic.js]: Same here.

File: lib/kinetic.js [hunk 3]
```diff
@@ -925,10 +935,11 @@ export class PutResponsePDU extends PDU {
  *                  {number} [options.clusterVersion = 0] - the cluster version
  * @returns {Kinetic} - message structure following the kinetic protocol
  */
-export class GetPDU extends PDU {
-    constructor(sequence, keyArg, options = {}) {
+class GetPDU extends PDU {
+    constructor(sequence, keyArg, optionsA) {
         super();
 
+        const options = optionsA ? optionsA : {};
```
Comment (michael-zapata) [inline lib/kinetic.js]: Here as well.

File: lib/kinetic.js [hunk 4]
```diff
@@ -1011,10 +1021,11 @@ export class GetResponsePDU extends PDU {
  *                  the operation.
  * @returns {Kinetic} - message structure following the kinetic protocol
  */
-export class DeletePDU extends PDU {
-    constructor(sequence, keyArg, options = {}) {
+class DeletePDU extends PDU {
+    constructor(sequence, keyArg, optionsA) {
         super();
 
+        const options = optionsA ? optionsA : {};
```
Comment (michael-zapata) [inline lib/kinetic.js]: And finally there.

File: lib/kinetic.js [hunk 5]
```diff
@@ -992,7 +1003,6 @@ export class GetResponsePDU extends PDU {
         return this;
     }
 }
-
```
Comment (michael-zapata) [inline lib/kinetic.js]: Why remove that line?

**Comment by AntoninCoulibaly** [comment]
@michael-zapata comments addressed!

**Pull request merged**